modules/helm/Makefile.build

```diff
@@ -0,0 +1,111 @@
+## Build chart $CHART_NAME from $CHART_TPL
+helm/chart/build: helm/chart/build/prepare helm/chart/build/deps helm/chart/build/package
+	@$(SELF) -s helm/chart/build/clean
+
+## Build chart $CHART_NAME from $CHART_TPL for all available $SEMVERSIONS
+helm/chart/build/all: helm/chart/build/prepare helm/chart/build/deps
+	@for version in $(SEMVERSIONS) ; do \
+		echo "Build chart $(CHART_NAME) version $$version"; \
+		$(SELF) -s helm/chart/build/package SEMVERSION=$$version; \
+	done
+	@$(SELF) -s helm/chart/build/clean
+
+## Alias for helm/chart/build/all. Depricated.
+helm/chart/build-all: helm/chart/build/all
+	@exit 0
+
+## Promote $CHART_FILE to $SEMVERSION
+helm/chart/promote/local: helm/chart/promote/local/prepare helm/chart/build/package
+	@$(SELF) -s helm/chart/build/clean
+
+## Promote $CHART from $CURRENT_VERSION to $SEMVERSION. ($REPO_ENDPOINT required)
+helm/chart/promote/remote: helm/chart/promote/remote/prepare helm/chart/build/package
+	@$(SELF) -s helm/chart/build/clean
+
+## Alias for helm/chart/publish/all. WARNING: Eventually will became functional equal to helm/chart/publish/one
```

Comment:
functionally equal

Reply:
As long as we're nitpicking: "become functionally equal"